src/components/HelloWorld.vue

```diff
@@ -0,0 +1,44 @@
+<script setup>
```

Comment:
Why do we need all of these files? I think we can delete this default vue greetings